docs/pages/reference/networking.mdx

```diff
@@ -36,21 +36,20 @@ following use cases:
 </TabItem>
 <TabItem scope={["cloud", "team"]} label="Cloud-Hosted">
 
-All Teleport services (e.g., the Application Service and Database Service) have
-an optional `public_addr` property that you can modify in each service's
-configuration file. The public address can take an IP or a DNS name. It can also
-be a list of values:
+For Teleport Enterprise (managed) the domain name and sub-domains
```

Comment:
We should indicate what "the domain name and sub-domains" refers to before we use a definite article like this.

Reply:
Please see update @ptgott